202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md

```diff
@@ -24,9 +24,108 @@ APPLICATION_STORE=US console propel:schema:copy
 console propel:install
 console transfer:generate
 ```
+4. Update the code that uses the product availabilities API response to handle the numeric strings. The API response changed in the following way: 
```

Comment:
Please mention that this appeared on 4.1.0 release. 
Also please update the composer require above, as the first valid v4 came only at 4.1.

Reply:
Also we need to mark 4.0.0 as a dead release, nobody should migrate there.